scripts/commands/build.js

```diff
@@ -33,18 +33,12 @@ async function buildRelease( currentBranchName ) {
 		return false;
 	}
 
-	// TODO - Verify (somehow) that we're not running with `start`.
-	/*
-	Some possible ideas for checking for `start`:
-
-	- Check for `build/*.map` files.
-	- The real build has a 948.js (not sure if this number changes) and start doesn't.
-	*/
 	if ( isDevBuild() ) {
-		error(
-			"You may have a dev build. Please make sure you aren't running 'npm start'"
+		return abortAndSwitchToBranch(
+			"You may have a dev build. Please make sure you aren't running 'npm start'",
+			'error',
+			currentBranchName
 		);
```

Comment:
I kept tripping over this error due to my `build` directory existing from some older build process - it may be worth sharing a command to work around this, even if it's just `rm -r build`.